src/main/java/org/qrush/brand/brand/helpers/BrandMapper.java

```diff
@@ -0,0 +1,22 @@
+package org.qrush.brand.brand.helpers;
+
+import org.qrush.brand.brand.dto.BrandDto;
+import org.qrush.brand.brand.models.Brand;
+import org.springframework.stereotype.Component;
+
+@Component
+public class BrandMapper {
+    public Brand toEntity(BrandDto brandDto) {
+        return Brand.builder()
+                .name(brandDto.getName())
+                .id(brandDto.getId())
+                .build();
+    }
+
+    public BrandDto toDTO(Brand brand) {
+        return BrandDto.builder()
+                .id(brand.getId())
+                .name(brand.getName())
+                .build();
+    }
```

Comment:
I'd move this logic into the BrandDto class, and put a `ToBrand` method on it. So the BrandDto would look like:

```java
public class BrandDto {
  public Brand ToBrand() {
    return BrandDto.builder()
      .id(brand.getId())
      .name(brand.getName())
      .build();
  }
}
```

which means you can remove some duplication and make these methods:

```java
public class BrandMapper {
    public Brand toEntity(BrandDto brandDto) {
        return brandDto.ToBrand();
    }

    public BrandDto toDTO(Brand brand) {
        return brandDto.ToBrand();
    }
```

Reply:
Updated to follow this method can you take a quick look to double check pls?